src/test/scala/mesosphere/marathon/integration/facades/MarathonFacade.scala

```diff
@@ -154,13 +154,18 @@ class MarathonFacade(
     * Connects to the Marathon SSE endpoint. Future completes when the http connection is established. Events are
     * streamed via the materializable-once Source.
     */
-  def events(): Future[Source[ITEvent, NotUsed]] = {
+  def events(eventsType: String*): Future[Source[ITEvent, NotUsed]] = {
 
     import EventUnmarshalling.fromEventStream
     val mapper = new ObjectMapper() with ScalaObjectMapper
     mapper.registerModule(DefaultScalaModule)
 
-    AkkaHttp().singleRequest(AkkaGet(s"$url/v2/events").withHeaders(AkkaAccept(AkkaMediaType.text("event-stream"))))
+    val filterQueryParam = eventsType
+      .map(eventType => s"event_type=$eventType")
+      .mkString("&")
+
+    AkkaHttp().singleRequest(AkkaGet(s"$url/v2/events?$filterQueryParam")
```

Comment:
This seems like it would break if eventType needs to be URL encoded. What if we used Akka http's `Uri` class?

```
Uri.parse(s"$url/v2/events").withQuery(Query("event_type" -> eventType))
```